plugins/modules/identity/keycloak/keycloak_user_info.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_user_info
+
+short_description: Allows obtaining Keycloak user information via Keycloak API
+
+version_added: 4.3.0
```

Comment:
This must be
```suggestion
version_added: 5.4.0
```
or later (this will be the next release).